gammapy/datasets/flux_points.py

```diff
@@ -448,17 +449,17 @@ def plot_spectrum(self, ax=None, kwargs_fp=None, kwargs_model=None):
         Parameters
         ----------
         ax : `~matplotlib.axes.Axes`
-            Axes to plot on.
+            Axes to plot on
         kwargs_fp : dict
-            Keyword arguments passed to `gammapy.estimators.FluxPoints.plot`.
+            Keyword arguments passed to `gammapy.estimators.FluxPoints.plot`
```

Comment:
maybe mention that these are matplotlib keywords to configure the plot style? otherwise it seems to be a bit of a rabbit hole to understand what should be contained in the awards

Reply:
Yes indeed... My suggestion was to simply add a couple of words like "Keyword arguments passed to `gammapy.estimators.FluxPoints.plot` to configure the plot style". not go into any details